eng/pipelines/mono/templates/xplat-job.yml

```diff
@@ -0,0 +1,111 @@
+parameters:
```

Comment:
This seems very similar to coreclr's xplat-job template, can we share them? Maybe move coreclr's to common.

Reply:
See answer below.

Reply:
https://github.com/dotnet/runtime/issues/2029